modules/core-updates/9.0.md

```diff
@@ -85,6 +85,40 @@ Some dependencies are still present but were upgraded which comes with their own
 - [Services translator, security.csrf.token_manager, session, session.flash_bag no longer public](https://github.com/PrestaShop/PrestaShop/pull/35069)
 - [Replace AdminSecurity annotation with attributes](https://github.com/PrestaShop/PrestaShop/pull/35212)
 
+### BO login and authorization migrated to Symfony
+
+The login page to the back-office has been migrated to Symfony, along ith it the system of authorization in the BackOffice is now also based on Symfony, it implies several things:
+- we no longer depend on the legacy `Context::$cookie`, the session is kept on the server side and very few data are kept on the browser side, for retro-compatibility we still populate the legacy cookie so that you can **read** it but modifying/writing it will likely no longer have the effect you would expect (if any)
```

Comment:
@jolelievre Jo, will this $context->cookie still be accessible or not? Because some things depend on it, For example - https://github.com/PrestaShop/ps_googleanalytics/blob/618ade8f9a424ad31968f6367ddf3e0be8c5f879/ps_googleanalytics.php#L131

If not, could you note some other way to store temporary data between sessions? ($session?)

Reply:
Hmm actually you can still write data in the cookie, it's just that the core no longer depends on it So for example you can't perform a BO login by manipulating the cookie anymore
But persisted data is still present in the cookie so it should be compatible for the modules, but you're right I'm gonna detail this a bit more

And regarding new possibilities to persist data, you can save attributes in the Symfony token That's what we did to handle:
- shop context
- employee session
- IP address checking

So it's worth detailing this as well I guess.